lib/canvas/live_events.rb

```diff
@@ -101,6 +101,23 @@ def self.get_submission_data(submission)
     }
   end
 
+  def self.get_attachment_data(attachment)
+    {
+      attachment_id: attachment.global_id,
+      user_id: attachment.global_user_id,
+      display_name: LiveEvents.truncate(attachment.display_name),
+      filename: LiveEvents.truncate(attachment.filename),
+      context_type: attachment.context_type,
+      context_id: attachment.context_id,
```

Comment:
Jut an FYI here: I added `context_type` and `context_id` on the attachment data as we need it on our end, but I'm not sure whether that's necessary? From re-reading the docs again, it looks like they might be part of the body metadata which should be available across all types of events?

I currently don't have access to an SQS queue where these messages should be getting delivered so it's hard to tell what might be required.